.speakeasy/workflow.lock

```diff
@@ -1,18 +1,20 @@
-speakeasyVersion: 1.637.2
+speakeasyVersion: 1.606.9
```

Comment:
Looks like you've got an outdated speakeasy version on your machine. Some version thrashing down below? Maybe need to re-zero and re-gen?

Reply:
I see, done.